paimon-core/src/main/java/org/apache/paimon/schema/SchemaManager.java

```diff
@@ -68,12 +70,14 @@ public class SchemaManager implements Serializable {
 
     private final FileIO fileIO;
     private final Path tableRoot;
+    private final Identifier identifier;
 
     @Nullable private transient Lock lock;
 
     public SchemaManager(FileIO fileIO, Path tableRoot) {
         this.fileIO = fileIO;
         this.tableRoot = tableRoot;
+        this.identifier = Identifier.fromPath(tableRoot, true);
```

Comment:
Can we just create a util method?
`Identifier  tableIdentifier`

Reply:
done.